Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -196,7 +196,7 @@ static o2::its::TrackITSExt cookSeed(const Point3Df& r1, Point3Df& r2, const Poi
   par[2] = sf;
 
   par[3] = 0.5 * (tgl12 + tgl23);
-  par[4] = (TMath::Abs(bz) < Almost0) ? Almost0 : crv / (bz * B2C);
+  par[4] = (TMath::Abs(bz) < Almost0) ? TMath::Sign(1 / o2::track::kMostProbablePt, crv) : crv / (bz * B2C);
```

Comment:
With bz=0 the sign of the crv estimate is random, do we need to propagate it to the par[4] assingment? I would simply assign ``1/o2::track::kMostProbablePt``

Reply:
@shahor02  In average, we should always have equal numbers of positive and negative charges.
So I think this effective randomisation should not hurt anything.

Reply:
Hmm, not sure what is the point of requirement for having equal number of + and - in the situation where the sign is not measured and is irrelevant for the tracking. BTW, you will not get the equal amount in this way, the exact ration will be determined by the residual misalignment.